src/core.jl

```diff
@@ -38,7 +38,7 @@ macro required(keywords...)
     for keyword in keywords
         push!(blocks, quote
             err_msg = string($(string(keyword)), " is required")
-            $(esc(keyword)) === nothing && @error(err_msg)
```

Comment:
Why did you remove `@` before the `error`? There is no `error` in Julia 1.0.0, only macro...